src/codeu/chat/client/ClientUser.java

```diff
@@ -44,19 +45,21 @@ public ClientUser(Controller controller, View view) {
     this.view = view;
   }
 
-  // Validate the username string
-  static public boolean isValidName(String userName) {
-    boolean clean = true;
-    if (userName.length() == 0) {
-      clean = false;
-    } else {
 
-      // TODO: check for invalid characters
+  // Makes sure the username/password combo doesn't contain any bad chars
+  static public boolean isValidUserInput(String username, String password) {
```

Comment:
I'd agree with Kevin in that you can probably do some checks on the username and password in this function rather than calling separate functions, especially since isValidInput() is a one liner.